### PR TITLE
Add release note for q3m5

### PR DESCRIPTION
**Pull request by sivagao**
### What is changed, added or deleted? (Required)

Add release note for Q3 Milestone 5 Release

### What is the related PR or file link(s)?

Reference link(s):

### CODE REVIEW

**Review by aylei — APPROVED** [review]
LGTM

File: release-notes.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ summary: Learn about the release notes of TiDB Cloud.
 
 This page lists the release notes of [TiDB Cloud](https://pingcap.com/products/tidbcloud).
 
+
+## September 11, 2020
+
+* Provide standard connection using public ip and traffic filters restriction
```
Comment (tennix) [inline release-notes.md:13]: Customers now can access TiDB using a public endpoint with traffic filters.

File: release-notes.md [hunk 2]
```diff
@@ -7,6 +7,13 @@ summary: Learn about the release notes of TiDB Cloud.
 
 This page lists the release notes of [TiDB Cloud](https://pingcap.com/products/tidbcloud).
 
+
+## September 11, 2020
+
+* Customers now can access TiDB using a public endpoint with traffic filters
+* Add time zone indicator at auto Backup settings dialog
```
Comment (lilin90) [inline release-notes.md:14]:
```suggestion
* Add the time zone indicator at the auto backup settings dialog
```

File: release-notes.md [hunk 3]
```diff
@@ -7,6 +7,13 @@ summary: Learn about the release notes of TiDB Cloud.
...
+* Fix broken invitation link when registration is not finished
```
Comment (lilin90) [inline release-notes.md:15]:
```suggestion
* Fix the broken invitation link when registration is not finished
```

**Review by lilin90 — COMMENTED** [review]
Rest LGTM

**Review by tennix — APPROVED** [review]
LGTM

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**